examples/scala-parallel-similarproduct/filterbyyear/build.sbt

```diff
@@ -7,6 +7,6 @@ name := "template-scala-parallel-similarproduct"
 organization := "io.prediction"
 
 libraryDependencies ++= Seq(
-  "io.prediction"    %% "core"          % "0.8.6" % "provided",
+  "io.prediction"    %% "core"          % "0.9.6" % "provided",
```

Comment:
@EmergentOrder isn't this wrong? This was the old aml 0.9.6. Don't we need 0.10.0-snapshot until release?